src/components/positions/PositionsCard.tsx

```diff
@@ -0,0 +1,170 @@
+import { useTheme } from '@/theme';
+
+import { RainbowPosition } from '@/resources/defi/PositionsQuery';
+import { GenericCard } from '../cards/GenericCard';
+import startCase from 'lodash/startCase';
+import { CoinIcon, RequestVendorLogoIcon } from '../coin-icon';
+import { AssetType, EthereumAddress } from '@/entities';
+import { useNavigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import { analyticsV2 } from '@/analytics';
+import { event } from '@/analytics/event';
+import { IS_ANDROID } from '@/env';
+import { capitalize } from 'lodash';
+
+type PositionCardProps = {
+  position: RainbowPosition;
+};
+
+type CoinStackToken = {
+  address: EthereumAddress;
+  type: AssetType;
+  symbol: string;
+};
+
+function CoinIconStack({ tokens }: { tokens: CoinStackToken[] }) {
+  const { colors } = useTheme();
+
+  return (
+    <Box flexDirection="row" alignItems="center">
+      {tokens.map((token, index) => {
+        return (
+          <Box
+            key={`availableNetwork-${token.address}`}
+            marginTop={{ custom: -2 }}
+            marginLeft={{ custom: index > 0 ? -8 : 0 }}
+            style={{
+              position: 'relative',
+              zIndex: tokens.length + index,
+              borderRadius: 30,
+              borderColor: index > 0 ? colors.transparent : colors.transparent,
+              borderWidth: 2,
+            }}
+          >
+            <CoinIcon
+              address={token.address}
+              size={16}
+              symbol={token.symbol}
+              type={token.type}
+            />
+          </Box>
+        );
+      })}
+    </Box>
+  );
+}
+
+export const PositionCard = ({ position }: PositionCardProps) => {
+  const { colors, isDarkMode } = useTheme();
+  const totalPositions =
+    (position.borrows?.length || 0) +
+    (position.deposits?.length || 0) +
+    (position.claimables?.length || 0);
+  const { navigate } = useNavigation();
+
+  const onPressHandler = useCallback(() => {
+    analyticsV2.track(event.positionsOpenedSheet, { dapp: position.type });
+    navigate(Routes.POSITION_SHEET, { position });
+  }, [navigate, position]);
+
+  const depositTokens: CoinStackToken[] = useMemo(() => {
+    let tokens: CoinStackToken[] = [];
+    position.deposits.forEach((deposit: any) => {
+      deposit.underlying.forEach((item: any) => {
+        tokens.push({
+          address: item.asset.asset_code,
+          type: AssetType.token,
+          symbol: item.asset.symbol,
+        });
+      });
+    });
+
+    return tokens;
+  }, [position]);
+
+  const positionColor =
+    position.dapp.colors.primary || position.dapp.colors.fallback;
+
+  return (
+    <Box width="full" height={{ custom: 117 }}>
+      <GenericCard
+        type={'stretch'}
+        onPress={onPressHandler}
+        color={colors.alpha(positionColor, 0.04)}
+        borderColor={colors.alpha(positionColor, 0.02)}
+        ignoreShadow={IS_ANDROID && isDarkMode}
```

Comment:
why ignore shadow if dark mode android?

Reply:
the shadow doesn't play well with the semi transparent background on android